src/routes/dbInit/import.ts

```diff
@@ -0,0 +1,553 @@
+// tslint:disable
+import fs from 'fs';
+import fetch  from 'node-fetch';
+const arr: any = [];
+const docs = ['smarter_balanced_ela_content_specification', 'air_deprecated_ela_claims_and_targets', 'air_deprecated_math_claims_and_targets', 'ccss_imported_from_digital_library',
+	'common_core_state_standards_for_ela', 'common_core_state_standards_for_mathematics', 'item_type_response_type', 'major_vs__additional_supporting__math_', 'major_vs__additional_supporting__math_', 'smarter_balanced_math_content_specification', 'norm_webb_s_depth_of_knowledge__dok__levels_of_cognitive_difficulty'];
+
+const claimArray: any = [1000];
+const claimArrayv: any = [1000];
+let docArr: any = [];
+let docNames: any = [];
+let jsonData;
+const claim = {
+	title: 'string',
+	claimNumber: 'string',
+	grades: ['string'],
+	subject: 'string',
+	description: 'string',
+	shortCode: 'string',
+	domain: 'string',
+	target: [
+		{
+			title: 'string',
+			shortCode: 'string',
+			description: 'string',
+			standards: [
+				'string'
+			],
+			stdDesc: [
+				'string'
+			],
+			DOK: [
+				'string'
+			],
+			DOKDesc: [
+				'string'
+			],
+			shortDOK: [
+				'string'
+			],
+			type: 'string',
+			clarification: 'string',
+			heading: 'string',
+			evidence: [
+				'string'
+			],
+			vocab: 'string',
+			tools: 'string',
+			stimInfo: 'string',
+			devNotes: 'string',
+			complexity: 'string',
+			dualText: 'string',
+			accessibility: 'string',
+			stem: [
+				'string'
+			],
+			taskDescription: [
+				'string'
+			],
+			taskModel: [
+				'string'
+			],
+			examples: [
+				'string'
+			],
+			rubrics: [
+				'string'
+			],
+			stimulus: [
+				'string'
+			],
+			shortStem: [
+				'string'
+			],
+
+		}
+	]
+};
+const claimv = {
```

Comment:
This and `claim` on line 13 were for debugging right? Are we going to have data that doesn't represent anything in the database like `"string"` when the value should be `null`?